scipy/cluster/tests/test_vq.py

```diff
@@ -233,10 +225,8 @@ def test_large_features(self, xp):
         data[:x.shape[0]] = x
         data[x.shape[0]:] = y
 
-        kmeans(xp.asarray(data), xp.asarray(2))
+        kmeans(xp.asarray(data), 2)
```

Comment:
I'll add a separate test for `kmeans*` with array input to this argument.

Reply:
addressed in [cc58124](https://github.com/scipy/scipy/pull/19682/commits/cc581243827c226caebdbbd8041ad08a224b1006)